.cargo/config

```diff
@@ -11,3 +11,10 @@ rustflags = [
     # Without this flag, we get miscompilation of floating point operations that cause the clipping region to be totally wrong
     "-C", "target-feature=-fp",
 ]
+
+[target.xtensa-esp32s2-none-elf]
+rustflags = [
+    # Enable the atomic codegen option for Xtensa
+    "-C",
+    "target-feature=+s32c1i",
+]
```

Comment:
please fix examples/mcu-board-support/esp32_s2_kaluga_1/cargo-config.toml similar to 9edd8d1dce7d2fdaea65d275dd3bdafe40a3a3bd